packages/babel-polyfill/package.json

```diff
@@ -8,7 +8,7 @@
   "repository": "https://github.com/babel/babel/tree/master/packages/babel-polyfill",
   "main": "lib/index.js",
   "dependencies": {
-    "core-js": "^1.0.1",
+    "core-js": "^2.0.0",
```

Comment:
Breaking changes for `babel-polyfill` users here only removing `RegExp.escape`, renaming `String` padding methods and removing obsolete `Array` generics. Other breaking changes in other places.